week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -26,6 +46,10 @@ The function must:
 */
 
 function remove(arr, index) {
+  let newIndex=arr.slice(0, index) // removes the item at the specified index
+  let newIndex2=arr.slice(index+1)
```

Comment:
A small thing, try to name the variables so that they convey the meaning of the data they store. Something `elementsBeforeIndex` and `elementsAfterIndex` for example, but feel free to use anything similar.